extensions/parquet/table/src/main/java/io/deephaven/parquet/table/transfer/PrimitiveArrayAndVectorTransfer.java

```diff
@@ -14,9 +14,13 @@
  * Used as a base class of transfer objects for arrays/vectors of primitive types.
  */
 abstract class PrimitiveArrayAndVectorTransfer<T, E, B extends Buffer> extends ArrayAndVectorTransfer<T, E, B> {
```

Comment:
The T=E condition doesn't hold true for the `DictEncodedStringArrayAndVectorTransfer` class because there we take a vector or array of strings and encode it as an array of integers.